sql-cloudant/README.md

```diff
@@ -62,6 +62,8 @@ cloudant.protocol|https|protocol to use to transfer data: http or https
 cloudant.host||cloudant host url
 cloudant.username||cloudant userid
 cloudant.password||cloudant password
+cloudant.useQuery|false|When enabled, for query not using index or view, _find will be used instead of _all_docs, some query predicates will be driven into datastore. However, RDD partition is ONE during _find, so parallel loading is not achieved
```

Comment:
my $ 0.02c, how about something like this... :

Clouding.optimizeQuery | false | When the optimizeQuery is enabled, the data source will look for ways to optimize the query execution by rewriting it or pushing down the execution to the data store. One example is the utilization of explicit _find with query predicates to be executed in the datastore instead of _all_docs. However, as RDD partition is ONE during _find, parallel loading is not achieved.